ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -133,17 +170,32 @@ cd $workdir
 
 if [ $? -ne 0 ]; then
    echo "*************** Running UFO failed ****************"
-   echo "Running EVA for diagnostic purposes"
+   RC=1
+   if [ $run_eva == YES ]; then
+       echo "Running EVA for diagnostic purposes"
+   else
+       exit 1
+   fi
 else
    echo "Running EVA"
```

Comment:
Should this echo here? seems it should only do so if the `if` is met below

Reply:
Correct.  Removing line.